src/Instrument/ClassLoading/CacheWarmer.php

```diff
@@ -83,6 +84,9 @@ public function warmUp()
                 );
 
                 $this->output->writeln(sprintf('<fg=green;options=bold>[OK]</>: <comment>%s</comment>', $path));
+            } catch (CircularWeavingException $e) {
```

Comment:
Here is the major idea -> when we catch CircularWeavingException -> there is no silencing, we kill the app, there is no continue with execution. 

Display in console, and re-throw exception